tools/CheckH5.py

```diff
@@ -0,0 +1,207 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import unittest
+import numpy as np
+import sys
+import yaml
+import os
+import h5py
+
+
+# Describes a particular comparison between H5 datasets or groups
```

Comment:
why not make all these comments docstrings? I'd like to (eventually) pull all these sort of files into Doxygen